sci-physics/spheno/spheno-3.3.8-r1.ebuild

```diff
@@ -3,29 +3,26 @@
 
 EAPI=8
 
-inherit toolchain-funcs
+inherit toolchain-funcs fortran-2
 
 MY_PN=SPheno
 MY_P=${MY_PN}-${PV}
 
 DESCRIPTION="SPheno stands for S(upersymmetric) Pheno(menology)"
 HOMEPAGE="https://spheno.hepforge.org/"
-SRC_URI="https://spheno.hepforge.org/downloads/?f=${MY_P}.tar.gz"
+SRC_URI="https://spheno.hepforge.org/downloads/?f=${MY_P}.tar.gz -> ${P}.tar.gz"
 S="${WORKDIR}/${MY_P}"
 
 LICENSE="all-rights-reserved"
 RESTRICT="bindist mirror"
 SLOT="0"
 KEYWORDS="~amd64"
 
-DEPEND="virtual/fortran"
-RDEPEND="${DEPEND}"
-
 PATCHES=( "${FILESDIR}"/${P}-gfortran.patch )
 
 src_compile() {
 	# single thread force needed since fortan mods depend on each other
-	export MAKEOPTS=-j1
+	export MAKEOPTS="$MAKEOPTS -j1"
 	tc-export FC AR
 	emake AR="${AR}" F90="${FC}"
 }
```

Comment:
```
src_compile() {
	# single thread force needed since fortan mods depend on each other
	tc-export FC AR
	emake -j1 AR="${AR}" F90="${FC}"
}
```

Reply:
It also seems a bit double to both `tc-export` and set the variables explicitly in the `emake` command.

Reply:
Normally `fortran-2` would set it but we need to overwrite `F90` by `FC`.

Reply:
But in this case it was a single PR adding AR https://github.com/gentoo/sci/pull/1162
Maybe it should be `${AR}` instead of `$(AR)` in the patch?

Reply:
Either way I am testing it now.